test/jobs/publish_apple_job_test.rb

```diff
@@ -3,9 +3,9 @@
 describe PublishAppleJob do
   let(:episode) { create(:episode, prx_uri: "/api/v1/stories/87683") }
   let(:podcast) { episode.podcast }
-  let(:feed) { create(:feed, podcast: podcast, slug: "adfree") }
+  let(:feed) { podcast.default_feed }
   let(:private_feed) { create(:private_feed, podcast: podcast) }
-  let(:apple_config) { create(:apple_config, podcast: podcast, public_feed: feed, private_feed: private_feed) }
+  let(:apple_config) { create(:apple_config, feed: private_feed) }
```

Comment:
So common pattern in the test refactor... the public `:feed` is now just the `podcast.default_feed`.  And the `:apple_config` only gets the private feed association.

Similarly, some method args change from the public-feed to the private-feed, since now it's the `private_feed.publish_apple? == true`.